mmsplice/generic.py

```diff
@@ -16,37 +16,6 @@ def expit(x):
     return 1. / (1. + np.exp(-x))
 
 
-def get_var_side(var):
```

Comment:
Any substitution of this function? It is also useful later to annotate variants by their region. 
An alternative version for this is
```
def get_var_side(var):
    ''' Get exon variant side '''
    varstart, ref, alt, start, end, strand = var

    # for long insertion or deletion which can go out side of exon
    # CA->CAGG
    varend = varstart + max(len(ref), len(alt)) - 1

    # left normalization
    for i in range(min(len(ref), len(alt))):
        if ref[i] == alt[i]:
            varstart += 1
        else:
            break

    if strand == "+":
        if varstart < start-50:
            return "left intron"
        elif varend <= start+3 and varstart >= start-50:
            return "acceptor"
        elif varend > end+13:
            return "right intron"
        elif varend <= end+13 and varstart >= end-5:
            return "donor"
        else:
            return "exon"
    else:
        if varstart < start-13:
            return "right intron"
        elif varstart >= start-13 and varend <= start+5:
            return "donor"
        elif varend > end+50:
            return "left intron"
        elif varstart >= end-3 and varend <= end+50:
            return "acceptor"
        else:
            return "exon"    
```

Reply:
The original function is restored.